hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/resolver/TestMountTableResolver.java

```diff
@@ -704,4 +704,26 @@ public void testLocationCache() throws Exception {
     mountTable.removeEntry("/testlocationcache");
     mountTable.removeEntry("/anothertestlocationcache");
   }
+
+  @Test
+  public void testInvalidateCache() throws Exception {
```

Comment:
Add a comment describing what the case to test is about.

Reply:
Thanks for your comments. I have added some comments. Pending jenkins. Please review again.

Reply:
The high level description should go as a javadoc.
You can add the particular comments inline.

Reply:
Thanks again. New commit here. Pending jenkins.